db/partners_migrate/20210924155700_add_partner_user_id_to_partner_requests.rb

```diff
@@ -0,0 +1,6 @@
+# capture which partner user created the partner request
+class AddPartnerUserIdToPartnerRequests < ActiveRecord::Migration[6.1]
+  def change
+    add_column :partner_requests, :partner_user_id, :integer, null: true, default: nil
```

Comment:
Could we add a foreign key constraint here? So that it isn't possible to set a `partner_user_id` that doesn't match any record?

Reply:
I will try out the foreign constraint on the old data to make sure we do not get any errors when completing existing requests that do not have a partner_user_id and add it to the migration if there are no issues.

Reply:
After talking with @scooter-dangle, we'll remove the DB constraint and then add validation to ensure this is defined for newer records being created.